frontend/src/components/pages/ComponentLibrary.tsx

```diff
@@ -71,13 +155,20 @@ const ButtonExamples = () => {
 
 const ComponentLibrary = (): React.ReactElement => {
   return (
-      <div style={{ textAlign: "center", paddingTop: "20px" }}>
-        <h1>Component Library</h1>
-        <div style={{ height: "2rem" }} />
-        <ButtonExamples />
-        <MainPageButton />
-      </div>
-    );
-  };
+    <div style={{ textAlign: "center", paddingTop: "20px" }}>
+      <h1>Component Library</h1>
+      <div style={{ height: "2rem" }} />
+      <ButtonExamples />
+      <MainPageButton />
+      <QuestionCard
+        date={new Date()}
+        questionTitle="Question Title"
+        image=""
+        text="Question Text"
```

Comment:
not sure if you removed it for other reasons, but would be great if you can add back the long text from your screenshots so we can see the cool ellipsis effect :)